examples/advanced/edge/project.yml

```diff
@@ -0,0 +1,36 @@
+api_version: 3
+name: edge_example
+description: NVFlare edge project yaml file for example
+
+# For edge project, the participants are created automatically
+# and organized in a tree hierarchy.
+edge:
+  analyze: false
+  lcp_only: false
+  root_dir: .
```

Comment:
we can set this to be: ""/tmp/nvflare/workspaces", then we don't need setup_nvflare.sh?

Reply:
ah right, missed that, updated